lib/Subscribers/SubscriberPersonalDataEraser.php

```diff
@@ -44,7 +43,7 @@ private function eraseCustomFields($subscriber_id) {
   }
 
   private function anonymizeSubscriberData($subscriber) {
-    $subscriber->email = sprintf('deleted-%s@site.invalid', uniqid('', true));
+    $subscriber->email = sprintf('deleted-%s@site.invalid', bin2hex(random_bytes(12))); // phpcs:ignore
```

Comment:
How does this work on PHP 5.6? I don't see any polyfill added or the function `random_bytes` being defined...

Reply:
WP already includes a polyfill for `random_bytes` here https://github.com/WordPress/WordPress/tree/master/wp-includes/random_compat